types/feathersjs__express/feathersjs__express-tests.ts

```diff
@@ -1,8 +1,11 @@
 import feathers, { Application } from '@feathersjs/feathers';
-import feathersExpress, { original, rest, notFound, errorHandler } from '@feathersjs/express';
+import feathersExpress, * as express from "@feathersjs/express";
```

Comment:
Please keep the quotes consistent.

This will be Subject to change anyway. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/24406 will finally allow `import express from '@feathersjs/express'` again.

Reply:
Well, it would be great, but i think it's ok for now, because I got errors during the compilation if I use destructuring.